frontend/components/rhf-elements/input.tsx

```diff
@@ -4,7 +4,9 @@ import { styled, TextField } from '@mui/material';
 import { LoginFormValues } from '../landing-page/component/login-card';
 
 interface Props {
-    name: string; 
+    // TODO: Find a way to set name as string. The compiler say: Type 'string' is not assignable to type '"email" | "password"'. 
+    // name: string;
```

Comment:
I'm not sure to understand what you mean here?

Reply:
Le type de la props `name` doit être un string. Le compilateur met une erreur disant que le type doit être `"email" | "password"` vu que c'est les deux valeurs qu'on a mis à date. dans les deux input instanciés.